l10n_ar_stock/wizards/arba_cot_wizard.py

```diff
@@ -43,6 +46,31 @@ class ArbaCotWizard(models.TransientModel):
         string='Importe Neto',
     )
 
+    @api.constrains('patente_vehiculo', 'patente_acoplado')
+    def _constrain_check_format_patente(self):
+        formato_antiguo = r'^[A-Z]{2}\d{3}[A-Z]{2}$'  # LLNNNLL
+        formato_nuevo = r'^[A-Z]{3}\d{3}$'            # LLLNNN
+        patente_valida = False
```

Comment:
Si hay un error en la patente no sabemos si es la aclopado o la de vehiculo, o si son las dos.
Separar en dos, pero incializar juntas

```suggestion
     patente_vehiculo_valida = patente_acoplado_valida = False```